src/client/components/HealthCareApp.jsx

```diff
@@ -94,11 +107,16 @@ class HealthCareApp extends React.Component {
     const path = this.props.location.pathname;
     const formData = this.props.data;
     const sectionFields = this.props.uiState.sections[path].fields;
+    const noScrollPages = ['/household-information/financial-disclosure', '/household-information/annual-income'];
```

Comment:
hm. I feel like the solution is to instead fix the scroll so it scrolls to the right error instead of just not scrolling at all. I don't think this solution is good because it could be that the error could be above the fold, and the user would think they did everything right and just keep clicking the continue button and not see that an error occurred (not likely to happen on the financial disclosure page but could very well be true on the annual income page if there is a spouse and dependents). I would rather remove this change and add it as a later thing to fix in a different PR.